.github/workflows/check-sonarcloud-on-push.yml

```diff
@@ -0,0 +1,18 @@
+name: "Check with Sonarcloud"
+
+on:
+  push:
+
+concurrency:
+  group: schulportal-testautomatisierung-${{ github.ref_name }}
+  cancel-in-progress: true
+
+jobs:
+  sonarcloud:
+    uses: dBildungsplattform/dbp-github-workflows/.github/workflows/check-nest-test-sonarcloud.yaml@DBP-332-sonarcloud
```

Comment:
Change when https://github.com/dBildungsplattform/dbp-github-workflows/pull/6 is merged and version tag added.